.github/workflows/release.yml

```
@@ -286,18 +292,23 @@ jobs:
       run: |
         # Sign Debian package
         version=${{ needs.prereqs.outputs.version }}
-        mv out/linux/Packaging.Linux/Release/deb/gcm-linux_amd64.$version.deb .
-        debsigs --sign=origin --verify --check gcm-linux_amd64.$version.deb
+        case ${{ matrix.runtime }} in
+          *x64) arch=amd64 ;;
+          *arm64) arch=arm64 ;;
+          *arm) arch=armhf ;;
+        esac
+        mv out/linux/Packaging.Linux/Release/deb/gcm-linux_$arch.$version.deb .
+        debsigs --sign=origin --verify --check gcm-linux_$arch.$version.deb
 
         # Generate tarball signature file
         mv -v out/linux/Packaging.Linux/Release/tar/* .
-        gpg --batch --yes --armor --output gcm-linux_amd64.$version.tar.gz.asc \
-          --detach-sig gcm-linux_amd64.$version.tar.gz
+        gpg --batch --yes --armor --output gcm-linux_$arch.$version.tar.gz.asc \
+          --detach-sig gcm-linux_$arch.$version.tar.gz
```

I'd like us to move to an artifact name on Linux that follows the .NET RID format `linux-(arm|arm64|x64)` rather than the current 'arch' format `linux_(amd64|arm64|armhf)`. This means we can simplify here:

```suggestion
        mv out/linux/Packaging.Linux/Release/deb/gcm-${{ matrix.runtime }}.$version.deb .
        debsigs --sign=origin --verify --check gcm-${{ matrix.runtime }}.$version.deb

        # Generate tarball signature file
        mv -v out/linux/Packaging.Linux/Release/tar/* .
        gpg --batch --yes --armor --output gcm-${{ matrix.runtime }}.$version.tar.gz.asc \
          --detach-sig gcm-${{ matrix.runtime }}.$version.tar.gz
```